gen/bind_generator.jl

```diff
@@ -115,7 +115,7 @@ macro bind(sig::Expr, err::Union{String,Expr,Nothing} = nothing)
     push!(funclist, string(funcsig))
 
     # Determine the underlying C library to call
-    lib = startswith(string(cfuncname), r"H5(DO|LT|TB)") ? :libhdf5_hl : :libhdf5
+    lib = startswith(string(cfuncname), r"H5(DO|LT|TB|DS)") ? :libhdf5_hl : :libhdf5
```

Comment:
```suggestion
    lib = startswith(string(cfuncname), r"H5(DO|DS|LT|TB)") ? :libhdf5_hl : :libhdf5
```

Keep in alphabetical order.